services/workflows-service/src/business-report/dto/create-batch-business-report-body.dto.ts

```diff
@@ -0,0 +1,10 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class CreateBatchBusinessReportBodyDto {
+  @ApiProperty({
+    type: 'string',
+    format: 'binary',
+    description: 'CSV file for batch business report',
+  })
+  file!: Express.Multer.File;
+}
```

Comment:
what validation do you use in here?
can you enforce the mimetype and file size ?

Reply:
This was done by Daniel, so I don't have an answer to that. I just extracted the `type` to a param because I needed the whole body to be of `FormData`